### PR TITLE
update smoking-duration and comment works

### PR DESCRIPTION
**Pull request by AdrianaSche**

### CODE REVIEW

File: backend/src/main/java/de/adrianaschepers/shishabuddies/controller/UserController.java [hunk 1]
```diff
@@ -161,11 +161,9 @@ public UserController(UserService userService) {
...
             currentSetup.setSmokingDuration(newSetup.getSmokingDuration());
         }
-          /*  if(!currentSetup.getNumOfSmokedHeads().equals(newSetup.getNumOfSmokedHeads())){
-                currentSetup.setNumOfSmokedHeads(newSetup.getNumOfSmokedHeads());
-            }*/
-
-        //SetupEntity updatedSetupEntity = userService.saveSetup(currentSetup,authUser);
+       /* if(!currentSetup.getNumOfSmokedHeads().equals(newSetup.getNumOfSmokedHeads())){
+            currentSetup.setNumOfSmokedHeads(newSetup.getNumOfSmokedHeads());
+        }*/
```
Comment (jamarob) [inline backend/src/main/java/de/adrianaschepers/shishabuddies/controller/UserController.java:166]: Please delete

File: backend/src/main/java/de/adrianaschepers/shishabuddies/repo/SetupEntityRepository.java [hunk 2]
```diff
@@ -1,7 +1,15 @@
...
 import de.adrianaschepers.shishabuddies.model.SetupEntity;
+import de.adrianaschepers.shishabuddies.model.UserEntity;
 import org.springframework.data.jpa.repository.JpaRepository;
 
+import java.util.Optional;
+
 public interface SetupEntityRepository extends JpaRepository<SetupEntity,Long> {
+
+    //Optional<SetupEntity> findAllByUserEntity(UserEntity userEntity);
+    //Optional<SetupEntity> findBySmokingDuration();
```
Comment (jamarob) [inline backend/src/main/java/de/adrianaschepers/shishabuddies/repo/SetupEntityRepository.java:12]: please delete

**Review by tobi-spec — APPROVED** [review]

File: backend/src/main/java/de/adrianaschepers/shishabuddies/service/UserService.java [hunk 3]
```diff
@@ -56,7 +64,7 @@ public UserEntity createUser(UserEntity userEntity) {
     }
 
 
-
+//SETTINGS
```
Comment (jamarob) [inline backend/src/main/java/de/adrianaschepers/shishabuddies/service/UserService.java:67]:
```suggestion

```

File: backend/src/main/java/de/adrianaschepers/shishabuddies/service/UserService.java [hunk 4]
```diff
@@ -82,6 +88,8 @@ public SettingsEntity saveSettings(SettingsEntity settingsEntity, UserEntity aut
         throw new EntityNotFoundException("user not in db");
     }
 
+
+    //SETUP
```
Comment (jamarob) [inline backend/src/main/java/de/adrianaschepers/shishabuddies/service/UserService.java:92]:
```suggestion
```

File: backend/src/main/java/de/adrianaschepers/shishabuddies/service/UserService.java [hunk 5]
```diff
@@ -99,12 +107,27 @@ public SetupEntity updateSetup(SetupEntity setupEntity){
...
+        Optional<SetupEntity> allSetups = setupEntityRepository.findAllBySmokingDuration();
+        if(allSetups.isPresent()){
+            getAllSetups(authUser).get().getSmokingDuration()
+        }
 
-    public List<UserEntity> getAll() {
-        return userRepository.findAll();
+        return smokingDuration;
     }
+*/
```
Comment (jamarob) [inline backend/src/main/java/de/adrianaschepers/shishabuddies/service/UserService.java:130]: Don't pollute main with comments

File: frontend/src/pages/Login.js [hunk 6]
```diff
@@ -26,10 +26,14 @@ export default function Login(props) {
     event.preventDefault()
     login(credentials).catch(error => console.error(error))
   }
+  /*const handleRegister = event => {
+    return <Redirect to="/register"/>
+}*/
```
Comment (jamarob) [inline frontend/src/pages/Login.js:31]:
```suggestion
```

File: frontend/src/pages/Login.js [hunk 7]
```diff
@@ -48,7 +52,8 @@ export default function Login(props) {
         />
 
         <Button> log' mich ein!</Button>
-        <Button> registrieren</Button>
+        {/*<Button> registrieren</Button>*/}
+        {/*<Button type="button" id="myButton" onClick={handleRegister}> registrieren</Button>*/}
```
Comment (jamarob) [inline frontend/src/pages/Login.js:56]:
```suggestion
```

**Review by jamarob — COMMENTED** [review]
Delete the comments, otherwise lgtm 🚬

File: backend/src/main/java/de/adrianaschepers/shishabuddies/controller/UserController.java (hunk 1 above)
Comment (AdrianaSche) [reply backend/src/main/java/de/adrianaschepers/shishabuddies/controller/UserController.java:166]: thx :)

**Pull request merged**